### PR TITLE
V2Wizard: Hide first boot from review step (HMS-3582)

### PR DESCRIPTION
**Pull request by amirfefer**
Follow-up from #1960

### CODE REVIEW

File: src/store/hooks.ts [hunk 1]
```diff
@@ -39,3 +42,9 @@ export const useServerStore = () => {
   const oscap = useOscapData();
   return { ...oscap };
 };
+
+export const useBetaFlag = (flag: string): boolean => {
```
Comment (ezr-ondrej) [inline src/store/hooks.ts:46]: not sure it belongs in store hooks, but it's fine IMO ;)

**Review by ezr-ondrej — DISMISSED** [review]
I love `useBetaFlag` helper idea ;)

File: src/store/hooks.ts [hunk 2]
```diff
@@ -39,3 +42,9 @@ export const useServerStore = () => {
...
+  const firstbootFlag = useFlag(flag);
```
Comment (regexowl) [inline src/store/hooks.ts:47]: If I understand correctly, this hook will be re-usable outside of first boot, right? Could you give the const more universal name? Like `unleashFlag` or something like that?

**Review by regexowl — COMMENTED** [review]
Nice! Love the `useBetaFlag` hook idea. Can we please move it outside of the store hooks and into a separate `useBetaFlag.ts` file in utilities folder? (doesn't matter if it's the one under components or V2Wizard, we'll have to clean those up when we remove V1)

**Pull request merged**